### PR TITLE
Expiry times on wireless guests

### PR DESCRIPTION
**Pull request by JPonte**
#### APK
[Download build #10552](http://192.168.120.33:8080/job/Pull%20Request%20Builder/10552/artifact/build/artifact/wire-dev-PR1468-10552.apk)
[Download build #10563](http://192.168.120.33:8080/job/Pull%20Request%20Builder/10563/artifact/build/artifact/wire-dev-PR1468-10563.apk)
[Download build #10566](http://192.168.120.33:8080/job/Pull%20Request%20Builder/10566/artifact/build/artifact/wire-dev-PR1468-10566.apk)
[Download build #10567](http://192.168.120.33:8080/job/Pull%20Request%20Builder/10567/artifact/build/artifact/wire-dev-PR1468-10567.apk)
[Download build #10568](http://192.168.120.33:8080/job/Pull%20Request%20Builder/10568/artifact/build/artifact/wire-dev-PR1468-10568.apk)
[Download build #10572](http://192.168.120.33:8080/job/Pull%20Request%20Builder/10572/artifact/build/artifact/wire-dev-PR1468-10572.apk)
[Download build #10584](http://192.168.120.33:8080/job/Pull%20Request%20Builder/10584/artifact/build/artifact/wire-dev-PR1468-10584.apk)
[Download build #10586](http://192.168.120.33:8080/job/Pull%20Request%20Builder/10586/artifact/build/artifact/wire-dev-PR1468-10586.apk)
[Download build #10588](http://192.168.120.33:8080/job/Pull%20Request%20Builder/10588/artifact/build/artifact/wire-dev-PR1468-10588.apk)
[Download build #10592](http://192.168.120.33:8080/job/Pull%20Request%20Builder/10592/artifact/build/artifact/wire-dev-PR1468-10592.apk)
[Download build #10595](http://192.168.120.33:8080/job/Pull%20Request%20Builder/10595/artifact/build/artifact/wire-dev-PR1468-10595.apk)

### CODE REVIEW

File: app/src/main/scala/com/waz/zclient/connect/SendConnectRequestFragment.scala [hunk 1]
```diff
@@ -83,10 +88,30 @@ class SendConnectRequestFragment extends BaseFragment[SendConnectRequestFragment
...
+              if (!isWireless)
+                showConnectButtonInsteadOfFooterMenu()
+            }
+            override def onRightActionClicked(): Unit = {
+              if (removeMemberEnabled)
+                getContainer.showRemoveConfirmation(userToConnectId)
+            }
+          })
+        }
+    }
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/connect/SendConnectRequestFragment.scala]: Break up this signal back into it's component parts and set them individually - the `FooterMenuCallback` is also set in `onViewCreated` and should be left there. On click we should check the value/head of the signal and respond accordingly

File: app/src/main/scala/com/waz/zclient/connect/SendConnectRequestFragment.scala [hunk 2]
```diff
@@ -125,6 +166,11 @@ class SendConnectRequestFragment extends BaseFragment[SendConnectRequestFragment
   override def onViewCreated(view: View, savedInstanceState: Bundle): Unit = {
     userNameView
     userHandleView
+    guestIndicator
+    guestIndicatorTimer
+
+    val color = if (themeController.isDarkTheme) R.color.wire__text_color_primary_dark_selector else R.color.wire__text_color_primary_light_selector
+    guestIndicatorIcon.foreach(_.setImageDrawable(GuestIcon(color)))
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/connect/SendConnectRequestFragment.scala]: may be better to listen to a signal of the theme

**Comment by someoneEsle** [comment]
- [ ] Bug 1
Although the +connect button on a wireless guest's details page is not visible, it is clickable and it allows you to connect to them

**Comment by someoneEsle** [comment]
- [ ] Bug 2
There is no devices tab

**Comment by someoneEsle** [comment]
- [ ] Bug 3 C662631
I can't see that wireless guests have been kicked out after they expire

**Comment by someoneEsle** [comment]
- [ ] bug 4
The profile picture for the wireless guest is sometimes so high that it is cut off by the details-devices bar

**Comment by someoneEsle** [comment]
- [ ] Bug 5
the +connect button is really small -- check with design

**Comment by someoneEsle** [comment]
- [ ] Bug 6
After I tap "create link" I am taken back to the conversation view

**Pull request merged**